src/test/java/elizCurtnazar/tests/tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,120 @@
+package elizCurtnazar.tests.tema6;
+
+import elizCurtnazar.driver.BrowserManager;
+import elizCurtnazar.utils.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HerokuAlerts {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToHerokuHomePage();
+        openAlertsPage();
+        openFirstAlert();
+        acceptFirstAlert();
+        openSecondAlert();
+        acceptAlert();
+        openThirdAlert();
+        acceptThirdAlert();
+        try {
+            openFakeAlertsPage();
+            System.out.println("Am intrat in blocul de try.");
+
+        } catch (NoSuchElementException e) {
+            System.out.println("Am intrat pe blocul de catch");
+            FileUtils.takeScreenshot(driver, "Fake Alert page");
+            //alta variante de rezolvare
+//            File file = driver.getScreenshotAs(OutputType.FILE);
+//            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza.png");
+//            FileUtils.copyFile(file, destinationFile);
+
+        } finally {
+            driver.close();
+            System.out.println("Am inchis pagina de pe blocul de finally");
+        }
+        closeBrowser();
```

Comment:
Replace driver.close from finally block to closeBrowser() and remove this closeBrowser from this line. Thanks!

Reply:
done